### PR TITLE
Feature/orca 89

### PR DESCRIPTION
**Pull request by Quarriers89**
## Summary of Changes

Created new pages for future Operator documents.

Addresses [ORCA-89: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/ORCA-89)

## Changes

* Created new pages for future Operator documents.
* ...

## Type of change

- [X] This change requires a documentation update

## Checklist:

- [X] My code follows the style guidelines of this project

- [X] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [X] User documentation

### CODE REVIEW

File: website/docs/operator/collection-configuration.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+id: collection-configuration
+title: Collection Configuration
+description: Provides documentation for Operators to configure a collection 
+using the Cumulus dashboard.
+---
+
+This document in under construction.
```
Comment (bhazuka) [inline website/docs/operator/collection-configuration.md:8]: Did you want to pull the information from the https://nasa.github.io/cumulus-orca/docs/developer/deployment-guide/deployment-with-cumulus#collection-configuration page and migrate to here?

File: website/docs/operator/data-recovery.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+id: data-recovery
+title: Data Recovery
+description: Provides documentation for Operators to recover missing data.
+---
+
+This document in under construction.
```
Comment (bhazuka) [inline website/docs/operator/data-recovery.md:7]: Did you want to add information on Data Recovery? Maybe something from https://wiki.earthdata.nasa.gov/display/PCESA/Disaster+Recovery+Operator+Guide or here https://wiki.earthdata.nasa.gov/display/LPCUMULUS/LPCumulus+Notes+on+Disaster+Recovery

**Review by bhazuka — COMMENTED** [review]
Everything looks ok to me but wondering if you wanted to add content to the pages or merely stub them out.

File: website/docs/operator/collection-configuration.md (hunk 1 above)
Comment (Quarriers89) [reply website/docs/operator/collection-configuration.md:8]: Will leave stubbed out, put in a card for content on this page, and work with Operators to develop content.

File: website/docs/operator/data-recovery.md (hunk 2 above)
Comment (Quarriers89) [reply website/docs/operator/data-recovery.md:7]: Will leave stubbed out, put in a card for content on this page, and work with Operators to develop content.

**Pull request merged**